templates/web/site-config.ftl

```diff
@@ -48,6 +48,7 @@
     <script src="/studio/static-assets/libs/js-cache/cache.js?version=${UIBuildId!.now?string('Mddyyyy')}"></script>
 
     <script>
+        $('html').attr('lang', CStudioAuthoringContext.lang);
```

Comment:
Write it directly on the tag.

```ftl
<html lang=“${ftlVarWhereValue}”>
```

Reply:
If not an FTL var, set it in studio-context.ftl where the CstudioAuthoringContext var is populated in the first place.